crates/sui-protocol-config/src/lib.rs

```diff
@@ -1434,6 +1442,7 @@ impl ProtocolConfig {
                 let mut cfg = Self::get_for_version_impl(version - 1, chain);
                 cfg.feature_flags.simple_conservation_checks = true;
                 cfg.max_publish_or_upgrade_per_ptb = Some(5);
+                cfg.feature_flags.loaded_child_object_format_type = true;
```

Comment:
you probably need to bump the existing code here to v24, and add 23 to include only your flag, or this will be difficult to pick to the release branch.